src/components/dismiss-button/dismiss-button.spec.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import TestRenderer from 'react-test-renderer';
+import { shallow, mount } from 'enzyme';
+import DismissButton from './dismiss-button.component';
+import { DismissButtonStyle, LinkStyle } from './dismiss-button.style';
+import 'jest-styled-components';
+import OptionsHelper from '../../utils/helpers/options-helper/options-helper';
+import classicTheme from '../../style/themes/classic';
+import { assertStyleMatch } from '../../__spec_helper__/test-utils';
+
+function render(props) {
+  return TestRenderer.create(<DismissButtonStyle { ...props } />);
+}
+
+describe('DismissButton', () => {
+  describe('when rendered', () => {
+    it('should render correctly', () => {
+      expect(shallow(<DismissButton />)).toHaveLength(1);
+    });
+  });
+
+  describe('when render with no additional props', () => {
+    OptionsHelper.messages.forEach((variant) => {
+      it(`should match snapshot for ${variant}`, () => {
+        const wrapper = render({ variant });
+        expect(wrapper).toMatchSnapshot();
```

Comment:
It's better to use `assertStyleMatch` here, as variant is responsible for a single css style